core/src/css/component/filter-dropdown-multiselect.component.scss

```diff
@@ -56,7 +56,9 @@
 		position: relative;
 		left: -1px;
 		top: -1px;
+		width: fit-content;
 		min-width: 220px;
+		max-width: 300px;
```

Comment:
What will happen if the text is too long? Maybe make sure it is truncated with ellipsis (...)? 

See for instance filter-dropdown.component.scss